ateam_msgs/msg/World.msg

```diff
@@ -0,0 +1,12 @@
+float32 current_time
```

Comment:
Should this be a `builtin_interfaces/Time` field instead?

Reply:
True true, forgot that was a thing